lua/lspconfig/configs/buflsp.lua

```diff
@@ -0,0 +1,21 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+
+    cmd = { 'buf', 'beta', 'lsp' },
+    filetypes = { 'proto' },
+    root_dir = function(fname)
+      return require('lspconfig.util').root_pattern('buf.work.yaml', '.git')(fname)
```

Comment:
`buf.work.yaml` seems to be old and the new preferred file is `buf.yaml`.

https://buf.build/docs/configuration/v2/buf-yaml/

Reply:
thanks for the suggestion, replaced :)